it/uk/gov/hmrc/fileupload/UploadFileIntegrationSpec.scala

```diff
@@ -46,11 +41,8 @@ class UploadFileIntegrationSpec extends IntegrationSpec with EnvelopeActions wit
       And("FileScanned")
       sendCommandMarkFileAsClean(MarkFileAsClean(envelopeId, fileId, fileRefId))
 
-      And("I have a valid file attached to the request body")
-      val data = "{}".getBytes
-
-      When(s"I invoke PUT envelope/$envelopeId/files/$fileId/$fileRefId")
-      val response: WSResponse = upload(data, envelopeId, fileId, fileRefId)
+      When(s"StoreFile($envelopeId, $fileId, $fileRefId, 123L) command is sent")
+      val response: WSResponse = sendCommandStoreFile(StoreFile(envelopeId, fileId, fileRefId, 123L))
 
```

Comment:
Why are we sending the command here? Do we not want to still test the upload endpoint?